src/renderer/component/speechUpload/view.jsx

```diff
@@ -152,9 +82,7 @@ class SpeechUpload extends React.PureComponent {
         <div
           className="card__content"
           style={
-            this.state.thumbnailUploadStatus === "sending"
-              ? null
-              : { display: "none" }
+            this.props.uploadStatus === "sending" ? null : { display: "none" }
```

Comment:
You should probably just not render this element at all, rather than rendering it and hiding it.